host/sgx/sgxquote.c

```diff
@@ -125,18 +163,213 @@ static bool _load_sgx_dcap_ql(void)
     return (_module != NULL);
 }
 
+static void _load_quote_ex_library_once(void)
+{
+    bool* tmp_mapped = NULL;
+    oe_uuid_t* tmp_uuid = NULL;
```

Comment:
Spell out "tmp" or remove it, per guidance in https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md to avoid abbreviations in names.

Reply:
Got it.